lightning-invoice/src/utils.rs

```diff
@@ -1117,9 +1115,9 @@ mod test {
 	#[cfg(feature = "std")]
 	fn test_multi_node_hints_has_htlc_min_max_values() {
 		let mut chanmon_cfgs = create_chanmon_cfgs(3);
-		let seed_1 = [42 as u8; 32];
-		let seed_2 = [43 as u8; 32];
-		let cross_node_seed = [44 as u8; 32];
+		let seed_1 = [42_u8; 32];
+		let seed_2 = [43_u8; 32];
+		let cross_node_seed = [44_u8; 32];
```

Comment:
I think this syntax *may* not be preferred by some but honestly is a little better IMO as there's no ambiguity around operator precedence for casts (plus it's shorter lol).

Reply:
Ok should run with this restriction: https://rust-lang.github.io/rust-clippy/master/index.html#separated_literal_suffix

For some reason forgot the no underscore syntax. Prefer that 👍